packages/shared/src/graphql/users.ts

```diff
@@ -77,6 +77,23 @@ export const USER_READING_HISTORY_QUERY = gql`
   }
 `;
 
+interface ReadHistorySource {
+  image: string;
+}
+
+interface ReadHistoryPost {
```

Comment:
I think it's better to pick the right properties from the `Post` interface rather than redeclare them